diff --git a/c++/src/kj/CMakeLists.txt b/c++/src/kj/CMakeLists.txt
index 795ea11b..b0bf8966 100644
--- a/c++/src/kj/CMakeLists.txt
+++ b/c++/src/kj/CMakeLists.txt
@@ -75,7 +75,7 @@ add_library(CapnProto::kj ALIAS kj)
 target_compile_features(kj PUBLIC cxx_generic_lambdas)
 
 if(UNIX AND NOT ANDROID)
-  target_link_libraries(kj PUBLIC pthread)
+  target_link_libraries(kj PUBLIC pthread rt)
 endif()
 #make sure the lite flag propagates to all users (internal + external) of this library
 target_compile_definitions(kj PUBLIC ${CAPNP_LITE_FLAG})
